upup/pkg/fi/cloudup/gce/wrappers.go

```diff
@@ -98,15 +96,10 @@ func ListManagedInstances(c GCECloud, igm *compute.InstanceGroupManager) ([]*com
 	//		googleapi.Field("items/metadata/items[key='instance-template']"),
 	//	)
 
-	var instances []*compute.ManagedInstance
-	err := c.Compute().InstanceGroupManagers.ListManagedInstances(project, zoneName, igm.Name).Pages(ctx,
-		func(page *compute.InstanceGroupManagersListManagedInstancesResponse) error {
-			instances = append(instances, page.ManagedInstances...)
-			return nil
-		})
+	req := c.Compute().InstanceGroupManagers.ListManagedInstances(project, zoneName, igm.Name)
+	resp, err := req.Do()
```

Comment:
As someone unfamiliar with the GCE api, I see that theres a [PageToken](https://pkg.go.dev/google.golang.org/api/compute/v1?tab=doc#InstanceGroupManagersListManagedInstancesCall.PageToken) for ListManagedInstances but I'm having trouble finding example code that uses it so I can't tell if we should continue to use pagination or not. Do you think that pagination is no longer necessary?

Reply:
It is unlikely in most cases that it would be needed, but you're right fixing...

Reply:
Well this is odd - it looks like the Pages method isn't defined there?  I can try to dig around to find out why but (1) do we agree there is no Pages method and (2) did we figure out a reason why it might not be needed?